src/data/dialControlParmeters.tsx

```diff
@@ -0,0 +1,106 @@
+import IDialControlParameters from "../interfaces/IDailControlParameters"
+import getDefaultContextWrapperValues from "../hooks/getDefaultContextWrapperValues";
+
+const defaultValues = getDefaultContextWrapperValues();
+
+function createParametersObject(
+    min: number,
+    max: number,
+    title: string,
+    localStorageKey: string,
+    defaultValue : number,
+    factor : number = 1
+     ) : IDialControlParameters{
+         return {
+             min,
+             max,
+             title, localStorageKey,
+             defaultValue,
+             factor
+         }
+}
+
+export const envelopeAttackParameters = createParametersObject(
+    1,
+    2000,
+    "ATTACK",
+    "envelopeAttackMs",
+    defaultValues.envelopeParameters.decayMs,
+)
+
+export const envelopeDecayParameters = createParametersObject(
+    1,
+    2000,
+    "DECAY",
+    "envelopeDecayMs",
+    defaultValues.envelopeParameters.decayMs,
+) 
+
+export const envelopeSustainParameters  = createParametersObject(
+    0,
+    1000,
```

Comment:
Why is this 0 to 1000? Sustain can technically only go from 0 to 1.

Reply:
I haven't figured out how to get the knob to play nice with non-integers